tests/Wallabag/CoreBundle/Helper/DownloadImagesTest.php

```diff
@@ -61,8 +61,8 @@ public function testProcessHtmlWithBadImage()
     public function singleImage()
     {
         return [
-            ['image/pjpeg', 'jpeg'],
-            ['image/jpeg', 'jpeg'],
+            ['image/pjpeg', 'jpg'],
+            ['image/jpeg', 'jpg'],
```

Comment:
I think is safe and BC because it seems this is used only to generate new image files, so no impact on previously generated ones I think, if someone can confirm :)